gc/base/MemorySubSpaceGenerational.cpp

```diff
@@ -325,7 +325,12 @@ MM_MemorySubSpaceGenerational::counterBalanceContract(
 uintptr_t
 MM_MemorySubSpaceGenerational::releaseFreeMemoryPages(MM_EnvironmentBase* env)
 {
-	return _memorySubSpaceOld->releaseFreeMemoryPages(env);
+	uintptr_t releasedPages = 0;
+	if (_releaseNurseryPages) {
+		releasedPages += _memorySubSpaceNew->releaseFreeMemoryPages(env);
+	}
+	releasedPages += _memorySubSpaceOld->releaseFreeMemoryPages(env);
```

Comment:
not a big deal, but you could do it in the same line where we define releasedPages